test/focus-first-suggestion/AutosuggestApp.test.js

```diff
@@ -135,7 +148,7 @@ describe('Autosuggest with highlightFirstSuggestion={true}', () => {
           suggestionValue: 'Perl',
           suggestionIndex: 0,
           sectionIndex: null,
-          method: 'enter'
+          method: 'enter',
```

Comment:
I understand the commit hook reformatted these lines... Not your fault... Still, to make the history digging and reviewing easier, I think it's better to split reformatting out to its own commit.

Reply:
I appreciate the feedback. I agree. I've went ahead and done that with my latest push for what it's worth.